inc/altis_config/class-qm-output-html-altis-config.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Altis Config data output for HTML pages in the Query Monitor plugin.
+ */
+
+namespace Altis\Dev_Tools\Altis_Config;
+
+use function Altis\get_config;
+use function Altis\get_environment_type;
+use QM_Collector;
+use QM_Output_Html;
+
+class QM_Output_Html_Altis_Config extends QM_Output_Html {
+
+	public function __construct( QM_Collector $collector ) {
+		parent::__construct( $collector );
+		add_filter( 'qm/output/panel_menus', [ $this, 'panel_menu' ], 50 );
+	}
+
+	public function output() {
+		$config = get_config()['modules'] ?? [];
+
+		if ( ! $config ) {
+			echo esc_html_x( 'Something went wrong with displaying the Altis config settings.', 'Displaying Altis config info in the Query Monitor plugin', 'altis' );
+			return;
+		}
+		?>
+
+		<?php $this->before_tabular_output(); ?>
+
+		<caption>
+			<h2><?php echo esc_html( $this->name() ); ?></h2>
+		</caption>
+		<thead>
+		<tr>
+			<th><?php echo esc_html__( 'Module', 'altis' ); ?></th>
+			<th><?php echo esc_html__( 'Final Settings', 'altis' ); ?></th>
+			<th><?php echo esc_html__( 'Environment Settings', 'altis' ); ?></th>
+		</tr>
+		</thead>
+		<tbody>
+		<?php foreach ( $config as $module => $settings ) : ?>
+			<tr>
+				<td><?php echo esc_html( $module ); ?></td>
+				<td>
+					<?php printf( '<pre>%s</pre>', print_r( $settings, true ) ); ?>
+					<br/>
+					<p><?php echo esc_html__( 'JSON encoded', 'altis' ); ?></p>
+					<?php echo json_encode( $settings, true ); ?>
```

Comment:
I was thinking instead of printing out the PHP array we show the JSON encoded version rather than both. They show the same information but JSON is a bit more readable and has the copy/paste advantage.

The JSON output should also be wrapped in `<pre><code>%s</code></pre>` tags.

Reply:
Agreed.

Also, we want to use `JSON_PRETTY_PRINT` here so it's human-readable. It also needs to be `esc_html`'d